docs/config-file/v2.rst

```diff
@@ -214,6 +225,13 @@ Read the Docs includes some libraries like scipy, numpy, etc.
 That you can access to them by enabling this option.
 See :ref:`builds:The build environment` for more details.
 
+.. warning::
+
+  If you are using a :ref:`Conda <config-file/v2:conda>` environment
+  to manage build, you should not set `system_packages` to `True`,
+  as this setting will not have any effect, since the virtual
+  environment creation is managed by Conda.
```

Comment:
This can be written as

```
If you are using a :ref:`Conda <config-file/v2:conda>` environment to manage the build,
this setting will not have any effect, since the virtual environment creation is managed by Conda.
```